pkg/openstack/heat.go

```diff
@@ -154,6 +154,11 @@ func ReconcileHeat(ctx context.Context, instance *corev1beta1.OpenStackControlPl
 		instance.Spec.Heat.Template.HeatCfnAPI.DeepCopyInto(&heat.Spec.HeatCfnAPI.HeatCfnAPITemplateCore)
 		instance.Spec.Heat.Template.HeatEngine.DeepCopyInto(&heat.Spec.HeatEngine.HeatEngineTemplateCore)
 
+		if heat.Spec.DatabaseInstance == "" {
+			//heat.Spec.DatabaseInstance = instance.Name // name of MariaDB we create here
+			heat.Spec.DatabaseInstance = "openstack" //FIXME: see above
+		}
+
```

Comment:
This shouldn't be necessary as of:
https://github.com/openstack-k8s-operators/heat-operator/commit/0717d7d2ad5c91634caba192663497c5d4abc5e3

Just needs to bump heat-operator in openstack-operator to pick it up now.

Reply:
I think the kubebuilder default isn't applied when the CR is created by a go struct